.github/workflows/code_quality_check.yml

```
@@ -32,19 +27,21 @@ jobs:
           cache-dependency-path: |
             **/requirement.txt
             **/test-requirement.txt
-      - name: Install dependencies.
+
+      - name: Install Dependencies
         run: |
           python -m pip install --upgrade pip
           python -m pip install -r requirement.txt
           python -m pip install -r tests/test-requirement.txt
-      - name: Running Black.
-        uses: psf/black@stable
+
+      - name: Run Ruff Formatter (Check)
```

I would rather have the name denote the action, not the tool.

```suggestion
      - name: Checking code formatting.
```